packages/babel-preset-env/src/use-built-ins-plugin.js

```diff
@@ -119,22 +119,28 @@ export default function({ types: t }: { types: Object }): Plugin {
         return;
       }
 
-      addImport(path, "web.dom.iterable", this.builtIns);
+      addImport(path, "es.array.iterator", this.builtIns);
+      addImport(path, "es.string.iterator", this.builtIns);
+      addImport(path, "web.dom-collections.iterator", this.builtIns);
     },
 
     // Symbol.iterator in arr
     BinaryExpression(path) {
       if (path.node.operator !== "in") return;
       if (!path.get("left").matchesPattern("Symbol.iterator")) return;
 
-      addImport(path, "web.dom.iterable", this.builtIns);
+      addImport(path, "es.array.iterator", this.builtIns);
+      addImport(path, "es.string.iterator", this.builtIns);
+      addImport(path, "web.dom-collections.iterator", this.builtIns);
     },
 
     // yield*
     YieldExpression(path) {
       if (!path.node.delegate) return;
 
-      addImport(path, "web.dom.iterable", this.builtIns);
+      addImport(path, "es.array.iterator", this.builtIns);
+      addImport(path, "es.string.iterator", this.builtIns);
+      addImport(path, "web.dom-collections.iterator", this.builtIns);
```

Comment:
Should it also be added to `ForOfStatement`and other features which works with iterators?